dialog/about.go

```diff
@@ -0,0 +1,136 @@
+package dialog
+
+import (
+	"image/color"
+
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/canvas"
+	"fyne.io/fyne/v2/container"
+	"fyne.io/fyne/v2/dialog"
+	"fyne.io/fyne/v2/layout"
+	"fyne.io/fyne/v2/theme"
+	"fyne.io/fyne/v2/widget"
+)
+
+// ShowAbout opens a parallax about dialog using the app metadata along with the
+// markdown content and links passed into this method.
+func ShowAbout(content string, links []*widget.Hyperlink, a fyne.App, w fyne.Window) {
```

Comment:
Can we get a `NewAbout()` so it is possible to not have to recreate it each time?